pages/platform/kubernetes-k8s/installing-operator/guide.en-gb.md

```diff
@@ -0,0 +1,789 @@
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - "serviceaccounts"
+  - "services"  
+  verbs:
+  - "*"
+- apiGroups:
+  - "apps"
+  verbs:
+    - "*"
+  resources:
+  - "deployments"
+---
+# The service account used for the Pod where is the operator
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ovh-nginx-operator-sa
+  namespace: ovh-nginx-operator
+---
+# The affected authorisation to the service account
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ovh-nginx-operator-admin
+subjects:
+- kind: ServiceAccount
+  name: ovh-nginx-operator-sa
+  namespace: ovh-nginx-operator
+roleRef:
+  kind: ClusterRole
+  name: ovhnginxoperator-admin-role
+  apiGroup: ""
+---
+# The deployment for the operator
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ovh-nginx-operator
+  namespace: ovh-nginx-operator
+spec:
+  selector:
+    matchLabels:
+      app: ovh-nginx-operator
+  replicas: 1 
+  strategy:
+    type: Recreate 
+  template:
+    metadata:
+      labels:
+        app: ovh-nginx-operator
+    spec:
+      serviceAccountName: ovh-nginx-operator-sa
+      containers:
+      - name: operator
+        image: 56hkk1xk.gra7.container-registry.ovh.net/example/ovh-nginx-operator:1.0.0
+        imagePullPolicy: Always
+      imagePullSecrets:
+      - name: regcred
+```
+And apply the file to the Kubernetes cluster:
+```bash
+kubectl apply -f manifests/ovh-nginx-operator.yaml
+```
+
+Output should be like this:
+<pre class="console"><code>
+$ kubectl apply -f manifests/ovh-nginx-operator.yaml
+namespace/ovh-nginx-operator created
+clusterrole.rbac.authorization.k8s.io/ovhnginxoperator-admin-role created
+serviceaccount/ovh-nginx-operator-sa created
+clusterrolebinding.rbac.authorization.k8s.io/ovh-nginx-operator-admin configured
+deployment.apps/ovh-nginx-operator created
+
+$ kubectl get pods --namespace ovh-nginx-operator
+NAME                                  READY   STATUS    RESTARTS   AGE
+ovh-nginx-operator-5487958499-v9q46   1/1     Running   0          70s
+</code></pre>
+
+
+It's time to test the operator.
+As previously you can apply the CR of the `samples` folder:
+
+```bash
+kubectl apply -f manifests/samples/tutorials_v1_ovhnginxoperator.yaml
+```
+
+Output should be like this:
+<pre class="console"><code>
+$ kubectl apply -f manifests/samples/tutorials_v1_ovhnginxoperator.yaml
+ovhnginx.tutorials.ovhcloud.com/mynginx-sample created
+
+$ kubectl get pods --namespace test-ovh-nginx-operator
+NAME                                        READY   STATUS    RESTARTS   AGE
+mynginx-sample-ovh-nginx-65b64c6585-5ddpm   1/1     Running   0          44s
+
+$ kubectl get services --namespace test-ovh-nginx-operator
+NAME                       TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)        AGE
+mynginx-sample-ovh-nginx   LoadBalancer   10.3.175.10   152.228.169.172   80:30403/TCP   2m36s
+</code></pre>
+
+
+Update the CR like with the following content and apply it again:
```

Comment:
Now you can change the operator behavior and tell it to create two replicas instead of one. To do that, in the `samples` folder edit the file `tutorials_v1_ovhnginxoperator.yaml` with the following content: